server/controllers/gym/editGym.ts

```diff
@@ -0,0 +1,20 @@
+import { NextFunction, Response, Request } from 'express';
+// import Sequelize from 'sequelize';
+import Gym from '../../database/models/gyms';
+import { CustomError, gymEditSchema } from '../../utils';
+
+export default async function editGymData(req: Request, res: Response, next: NextFunction) {
```

Comment:
Here You need Check if the gym owner Requested a new logo > remove the old logo from **AWS** then upload a new image.